packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -57,6 +57,8 @@ import { CWTag } from './new_designs/cw_tag';
 import { CWThreadAction } from './new_designs/cw_thread_action';
 import { CWToggle, toggleDarkMode } from './new_designs/cw_toggle';
 import { CWUpvote } from './new_designs/cw_upvote';
+import { CWTypeaheadSelectList } from './new_designs/CWTypeaheadSelectList/CWTypeaheadSelectList';
```

Comment:
We have `index.ts` file to not have duplicated names in the path

```suggestion
import { CWTypeaheadSelectList } from './new_designs/CWTypeaheadSelectList';
```

Reply:
Updated!